Dockerfile.composer

```diff
@@ -0,0 +1,14 @@
+FROM composer:1.7.1
+
+ARG composer_dev_arg
+
+COPY composer.json \
+    composer.lock \
+    symfony.lock \
+    ./
+
+RUN composer --no-interaction install ${composer_dev_arg} --ignore-platform-reqs --no-autoloader --no-suggest --prefer-dist
+
+COPY src/ src/
```

Comment:
[I heard](https://github.com/symfony/recipes/blob/master/README.rst#copy-from-recipe-configurator) Symfony Flex could also write to `config/`?

Reply:
(but maybe not during these `composer` commands)

Reply:
Yeah, when adding a new dependency. Might need to mount `config/` to `composer` so that changes are captured if using Composer through the container.

Reply:
Added in f853fa9afd17bbe8faf8b0ac517dcd9709640fba (and tested by running `docker-compose run composer composer require orm`).